vcfa/resource_vcfa_content_library_item.go

```diff
@@ -40,11 +41,14 @@ func resourceVcfaContentLibraryItem() *schema.Resource {
 				ForceNew:    true,
 				Description: fmt.Sprintf("ID of the %s that this %s belongs to", labelVcfaContentLibrary, labelVcfaContentLibraryItem),
 			},
-			"file_path": {
-				Type:        schema.TypeString,
+			"files_paths": {
```

Comment:
Thought for consideration. Could we read the referenced files automatically? Similar like here:
https://registry.terraform.io/providers/vmware/vcd/latest/docs/resources/catalog_item#ovf_url-1

Reply:
I saw there are third party Go packages for that if we want to parse the OVF and find the files.

The `ovf_url` is not supported in VCFA (in other words, VCD Provider just tells the backend about the URL, VCD was doing the magic behind, but VCFA does not allow supplying an URL, one must feed all files directly)

Reply:
Got it. Then let's move forward with current approach.